lib/attachment_processors/image.rb

```diff
@@ -24,7 +25,7 @@ def resize_and_upload(width: 600, height: 400, tag: :thumb)
       tmp_img = Tempfile.new([SecureRandom.hex(18), ext])
 
       # resize it
-      img = Magick::Image.read(attachment.url).first
+      img = Magick::Image.from_blob(open(attachment.url).read).first
```

Comment:
For some reason rmagick wasn’t reading the image from URL locally, needed to read the raw data. Seen a few articles where certain versions of imagemagick dont support opening a file by url.